### PR TITLE
fix: restore missing .env files

### LINKED ISSUE #527: Restore missing files

### PR DESCRIPTION
**Pull request by JustinCast**
### GH Issue

closes #527 

### Steps to test
1. `make run` at the root directory

### Checklist
- [ ] I have added/updated tests to cover these changes.
- [X] The branch name, commit history and PR title follow [conventions](https://developers.eoscostarica.io/docs/open-source-guidelines#pull-request-general-guidelines).
- [X] I have taken into consideration any impact to site performance.

### CODE REVIEW

File: services/frontend/.env.production [hunk 1]
```diff
@@ -0,0 +1,8 @@
+REACT_APP_GRAPHQL_HTTP_URL=https://graphql-jungle.eosrate.io/v1alpha1/graphql
+REACT_APP_API_URL=https://jungle.eosio.cr
```
Comment (xavier506) [inline services/frontend/.env.production:2]: @JustinCast @kuronosec 

are we using jungle or mainnet in production?

Comment (JustinCast) [reply services/frontend/.env.production]: @xavier506  I updated the file. Please take another look

File: services/frontend/.env.production [hunk 2]
```diff
@@ -0,0 +1,8 @@
+REACT_APP_GRAPHQL_HTTP_URL=https://graphql.eosrate.io/v1/graphql
+REACT_APP_API_URL=https://bp.cryptolions.io
```
Comment (xavier506) [inline services/frontend/.env.production:2]: I think we can also use our own api 

https://api.eosio.cr

Comment (JustinCast) [reply services/frontend/.env.production]: added!

**Review by xavier506 — APPROVED** [review]

**Pull request merged**